HIRS_Utils/src/main/java/hirs/client/collector/DeviceInfoCollector.java

```diff
@@ -100,6 +105,66 @@ public DeviceInfoCollector(
         LOGGER.warn("default constructor called! Should only be done in test");
     }
 
+    private static String callPACCOR() throws CollectorException {
```

Comment:
nit: I believe our style guide says we should only capitalize the first letter of acronyms in camel case

Reply:
changed.